src/handleAction.js

```diff
@@ -1,19 +1,18 @@
 import isFunction from 'lodash/isFunction';
 import identity from 'lodash/identity';
 import isNil from 'lodash/isNil';
-import isSymbol from 'lodash/isSymbol';
+import includes from 'lodash/includes';
+import { ACTION_TYPE_DELIMITER } from './combineActions';
 
-export default function handleAction(type, reducers, defaultState) {
-  const typeValue = isSymbol(type)
-    ? type
-    : type.toString();
+export default function handleAction(actionType, reducers, defaultState) {
+  const actionTypes = actionType.toString().split(ACTION_TYPE_DELIMITER);
```

Comment:
Hm, I'm not sure. I think the code is a bit clearer without the special cases, and we don't know that performance will be an issue yet here (and I haven't seen it yet in a production app). Can we optimize when we know we'll need it and leave the code without special cases for now (for better readability)?